platform/pulp/app/models/storage.py

```diff
@@ -10,10 +11,98 @@
 from django.utils.deconstruct import deconstructible
 
 
+class FileContent(File):
```

Comment:
Curious why this wouldnt work:
https://docs.djangoproject.com/en/1.11/_modules/django/core/files/base/#File